kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -437,8 +425,8 @@ public void run() {
 
     @Override
     public long getDelay(TimeUnit unit) {
```

Comment:
`getDelayMs`?

Reply:
This implements `java.util.concurrent.Delayed#getDelay(TimeUnit)`. Can't rename.